docker-compose.yml

```diff
@@ -1,27 +1,40 @@
-version: '2'
+version: '3.8'
+
 services:
   postgres:
-    image: postgres:9.5-alpine
+    image: postgres:12-alpine
+    hostname: stringer-postgres
     restart: always
+    networks:
+      - stringer
     volumes:
-      - ~/stringer:/var/lib/postgresql/data
+      - ./postgres:/var/lib/postgresql/data
     environment:
-      - POSTGRES_PASSWORD=super_secret_password
+      - POSTGRES_PASSWORD=<PLEASE_ENTER_YOUR_PASSWORD>
```

Comment:
Personally I don't care too much for changes like these since the password will need to be changed anyway. If we want to make it so that the container won't launch without a password change then we should probably just not set this value

Reply:
I'm fine with cosmetic changes, and this seems like maybe an improvement. If there's a way to force the user to input it, though, so that they can't miss it, that seems even better.